clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf8
+
+
+import clinica.engine as ce
+from colorama import Fore
+
+class DeepLearningPrepareDataCLI(ce.CmdParser):
+
+    def define_name(self):
+        """Define the sub-command name to run this pipeline."""
+        self._name = 'deeplearning-prepare-data'
+
+    def define_description(self):
+        """Define a description of this pipeline."""
+        self._description = ('Prepare data generated Clinica for PyTorch with Tensor extraction:\n'
+                             'http://clinica.run/doc/Pipelines/DeepLearning_PrepareData/')
+
+    def define_options(self):
+        """Define the sub-command arguments."""
+        from clinica.engine.cmdparser import PIPELINE_CATEGORIES
+
+        # Clinica compulsory arguments (e.g. BIDS, CAPS, group_id...)
+        # Most of the time, you will want to read your pipeline inputs into
+        # a BIDS and/or CAPS directory. If your pipeline does not require BIDS input,
+        # simply remove the two lines involving the BIDS directory.
+        clinica_comp = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_COMPULSORY'])
+        clinica_comp.add_argument("caps_directory",
+                                  help='Path to the CAPS directory.')
+        clinica_comp.add_argument("tsv_file",
+                                  help='Path to the tsv file.')
```

Comment:
To remove:
```suggestion
```
This is actually done by `args.subjects_sessions_tsv ` via `self.add_clinica_standard_arguments() ` :)